### ISSUE TITLE
Badge request: Katacoda

### ISSUE THREAD

**Issue by adriens** [badges/shields#5758]
Hi, that would be great to have a Katacoda badge, I may consifer this as a social platform like GH/Twitter.

**Comment by PyvesB** [comment]
Hello @adriens ! 👋🏻 

Thanks for the suggestion. I'm not familiar with this service, what would such a badge display?

**Comment by adriens** [comment]
Hi,

it would display the number of scenarios. You can get a preview of what Katacoda provides by default:

https://github.com/adriens/katacoda-scenarios

![image](https://user-images.githubusercontent.com/5235127/97102048-4298ad00-16f6-11eb-8162-b1296921f6ed.png)

**Comment by PyvesB** [comment]
Okay, the badge would need to be slightly different to be consistent with Shields' offering and follow our guidelines (https://github.com/badges/shields/blob/master/CONTRIBUTING.md#badge-guidelines), but apart from that it should be fine.

Do you know if there's an API available to easily retrieve the number of scenarios?

**Comment by adriens** [comment]
I'm trying to look for if there is a public API I'll let you know 👍

**Comment by adriens** [comment]
I'm on it 🚀

**Comment by adriens** [comment]
I've just dropped en email to their support:

> Hi guys,
> I'd like to be able to get Katacoda badges from shields.io to embed some of my katacoda metrics on my projects/webpages.
> I've created a dedicated issue: https://github.com/badges/shields/issues/5758
> So the question is: do you have any public API that can deliver scores on:
> number of coursesnumber of scenariosother score centric metrics (nb of scenarios achieved, ...)
> I really love your platform, getting public APIs would be awesome... I could even write  a scenario on how to consume it ;-p
> Hope to see you soon on the GH ticket.
> Kind regards,Adrien
> 

now let's 🤞

**Comment by adriens** [comment]
No answer for now ⏸️

**Comment by adriens** [comment]
I have quoted this issue on an article I hava dedicated to Katacoda and its integrations
https://www.linkedin.com/posts/adrien-sales_katacoda-freedom-inspiration-activity-6728075471919157248-QFID

**Comment by adriens** [comment]
Things are on the good way 🚀 🥇 
![image](https://user-images.githubusercontent.com/5235127/98337208-61833180-205c-11eb-8d7c-430e9270790b.png)

**Comment by adriens** [comment]
https://www.linkedin.com/posts/adrien-sales_api-katacoda-badges-activity-6730381383874805760-shaX

**Comment by PyvesB** [comment]
Great, seems like the badge will be doable in a few months then. 😉

**Comment by adriens** [comment]
Badges rule ❣️

**Comment by ronan696** [comment]
> Great, seems like the badge will be doable in a few months then. 😉

Definitely! The Katacoda icon has also been added to the Simple Icons repo 🎉 
https://github.com/simple-icons/simple-icons/pull/3790

**Comment by PyvesB** [comment]
It looks like Katacoda has been sunset ([source](https://www.oreilly.com/online-learning/leveraging-katacoda-technology.html)). Will go ahead and close the issue.
